tools/serverpod_cli/lib/src/test_util/builders/model_source_builder.dart

```diff
@@ -11,12 +13,18 @@ class ModelSourceBuilder {
     fields:
       name: String
     ''');
-    yamlSourceUri = Uri(path: 'lib/src/model/example.yaml');
+    yamlSourcePath = 'lib/src/model/example.yaml';
     protocolRootPathParts = [];
+    moduleAlias = defaultModuleAlias;
+  }
+
+  ModelSourceBuilder withModuleAlias(String moduleAlias) {
+    this.moduleAlias = moduleAlias;
+    return this;
   }
 
   ModelSourceBuilder withFileName(String fileName) {
-    yamlSourceUri = Uri(path: 'lib/src/model/$fileName.yaml');
+    yamlSourcePath = 'lib/src/model/$fileName.yaml';
```

Comment:
Suggestion: Build the paths in this file using the path lib.

Might as well make sure it is all compatible with windows directly, especially since these are the builders for the tests.